Release 1.16.0 (#2627)
fluxcd/flux: CHANGELOG.md

## 1.16.0 (2019-11-22)

This is a feature release with minor new features. New flags
`--manifest-generation` and `--read-only` have been added to
`fluxctl install`.

This release also incorporates a few fixes and enhacements. Namely:
 * The pressure on the Kubernetes API server has been reduced when

>    Flux operatres in all namespaces.

Reviewer: ```suggestion
   Flux operates in all namespaces.
```

 * The error handling of manifest generation has been improved.

Additionally, the end-to-end testing infrastructure has been rewritten and
numerous new end-to-end tests have been added.

### Fixes

- Exclude the metrics APIs from resource discovery [fluxcd/flux#2606][]
- Parse image refs in HelmReleases with >2 elements [fluxcd/flux#2620][]
- Ignore timestamp labels during sorting and release of images [fluxcd/flux#2594][]

> - Security: Stop showing value of `GIT_AUTHKEY` in the logs [fluxcd/flux#2549][]

Reviewer: ```suggestion
- Security: Stop showing value of `GIT_AUTHKEY` in the `fluxctl` output [fluxcd/flux#2549][]
```

### Enhacements

> - Improve experience with .flux.yaml files

Reviewer: ```suggestion
- Improve experience with `.flux.yaml` files
```

Nitpick but imho more in line with our usage of ``
